### ISSUE TITLE
Sizing View Controllers

### ISSUE THREAD

**Issue by dtp5** [lbrendanl/SwiftSwipeView#6]
Hey Brendan, thanks for the repo. Your code works perfectly with .xibs, but when I try to substitute view controllers for .xibs, the sizes are all thrown off. The only code I changed was: 

This:
`var AVc :AViewController =  AViewController(nibName: "AViewController", bundle: nil);
        var BVc :BViewController =  BViewController(nibName: "BViewController", bundle: nil);
        var CVc :CViewController =  CViewController(nibName: "CViewController", bundle: nil);`

To this:
`var AVc =  storyboard!.instantiateViewControllerWithIdentifier("VC1") as! UIViewController
        var BVc =  storyboard!.instantiateViewControllerWithIdentifier("VC2") as! UIViewController
        var CVc =  storyboard!.instantiateViewControllerWithIdentifier("VC3") as! UIViewController`

![img_2706](https://cloud.githubusercontent.com/assets/6037116/7644011/be18ef32-fa53-11e4-84c0-03df311343c3.PNG)
![img_2707](https://cloud.githubusercontent.com/assets/6037116/7644012/c06b4e06-fa53-11e4-981e-1f4f19442a46.PNG)
![img_2708](https://cloud.githubusercontent.com/assets/6037116/7644013/c3c21364-fa53-11e4-82f0-2df573476d0b.PNG)

**Comment by lbrendanl** [comment]
@dtp5 were you able to get the original code working correctly? I just grabbed a fresh copy of master, added 3 view controllers to the storyboard and placed in ne Storyboard IDs to each view controller and then used the same three lines that you had, and it worked correctly for me.

**Comment by dtp5** [comment]
Yeah I added view controllers to your project and it ran fine. Only in my project are the sizes distorted. I've tried everything. One thing I'm thinking about is that I have an iPhone 6, and your project doesn't have native resolution on my phone. My project has native resolution, but the sizes are all off. Could that be it?

I've even copy+pasted view controllers from your project into mine and had the same problems.

**Comment by leighfurby** [comment]
I've been having the same issue, try as I might, the Views won't go edge to edge (5 fine, 6 has a gap) but the original Repo does this fine - it's so strange!
I've been hunting for strange settings in the plist and all sorts.

**Comment by LuAndreCast** [comment]
@lbrendanl I did the same above and was having the same issue at first. 
Resolve the issue by removing the "auto size classes" and by making sure that all my labels and items on the view controller had appropriate constraints.

**Comment by desduvauchelle** [comment]
I'm having the exact same issue as the initial post. Has a solution been found? It's driving me nuts. It's copy paste identical! 

PS: Thanks for the awesome repo despite the issue :)

**Comment by sudeepag** [comment]
Was there any solution to this problem? I'm facing the same issue.
